conventions/src/main/kotlin/io.opentelemetry.instrumentation.javaagent-testing.gradle.kts

```diff
@@ -93,7 +93,8 @@ class JavaagentTestArgumentsProvider(
     "-Dio.opentelemetry.javaagent.slf4j.simpleLogger.log.io.grpc.internal.ServerImplBuilder=INFO",
     "-Dio.opentelemetry.javaagent.slf4j.simpleLogger.log.io.grpc.internal.ManagedChannelImplBuilder=INFO",
     "-Dio.opentelemetry.javaagent.slf4j.simpleLogger.log.io.perfmark.PerfMark=INFO",
-    "-Dio.opentelemetry.javaagent.slf4j.simpleLogger.log.io.grpc.Context=INFO"
+    "-Dio.opentelemetry.javaagent.slf4j.simpleLogger.log.io.grpc.Context=INFO",
+    "-Dotel.java.experimental.span-attributes.copy-from-baggage.include=*"
```

Comment:
perhaps it would be better to list the keys that you use in the test here to avoid surprises when baggage is used in some other test